utils/network.go

```diff
@@ -182,9 +184,18 @@ func setupRoutes(hostVeth netlink.Link, result *current.Result) error {
 				LinkIndex: hostVeth.Attrs().Index,
 				Scope:     netlink.SCOPE_LINK,
 				Dst:       &ip.Address,
-			})
+			},
+		)
+
 		if err != nil {
-			return fmt.Errorf("failed to add route %v", err)
+			switch err {
+			case errFileExists:
+				// Route was already present on the host.
```

Comment:
I can imagine this going a few ways, namely:
- We ignore and assume the route is correct, OR
- We take a look and re-program if it's not the right route.

Should probably understand what are the scenarios where we'll get "route exists" errors.  I'm assuming it's just when a route to the exact `w.x.y.z/32` exists?

Do we hit this when running alongside say, another plugin that thinks it owns that IP address?